storage/src/lsm_tree/components/block.rs

```diff
@@ -16,13 +16,13 @@ use crate::{Error, Result};
 
 pub struct Block {
     /// Uncompressed entries data.
-    data: Bytes,
+    data: Vec<u8>,
     /// Restart points.
     restart_points: Vec<u32>,
 }
 
 impl Block {
-    pub fn decode(buf: Bytes) -> Result<Self> {
+    pub fn decode(buf: Vec<u8>) -> Result<Self> {
```

Comment:
I think `decode` can also use `&[u8]` now. 🤔 After all, there is always a copy needed at last.

Reply:
👌